Dockerfile

```diff
@@ -48,11 +52,12 @@ COPY --from=builder /che-operator/templates/keycloak-provision.sh /tmp/keycloak-
 COPY --from=builder /che-operator/templates/oauth-provision.sh /tmp/oauth-provision.sh
 COPY --from=builder /che-operator/templates/delete-identity-provider.sh /tmp/delete-identity-provider.sh
 COPY --from=builder /che-operator/templates/create-github-identity-provider.sh /tmp/create-github-identity-provider.sh
-COPY --from=builder /tmp/devfile-devworkspace-operator-*/deploy /tmp/devworkspace-operator/templates
-COPY --from=builder /tmp/che-incubator-devworkspace-che-operator-*/deploy /tmp/devworkspace-che-operator/templates
+COPY --from=builder /tmp/devworkspace-operator/templates/deploy /tmp/devworkspace-operator/templates
+COPY --from=builder /tmp/devworkspace-che-operator/templates/deploy /tmp/devworkspace-che-operator/templates
 
-# apply CVE fixes, if required
-RUN microdnf update -y librepo libnghttp2 && microdnf install httpd-tools && microdnf clean all && rm -rf /var/cache/yum && echo "Installed Packages" && rpm -qa | sort -V && echo "End Of Installed Packages"
+# install dnf so we can update RPMs w/ latest CVE fixes
+# install httpd-tools for /usr/bin/htpasswd
+RUN microdnf install -y dnf httpd-tools && dnf -y -q update && dnf -y -q clean all && rm -rf /var/cache/yum && echo "Installed Packages" && rpm -qa | sort -V && echo "End Of Installed Packages"
```

Comment:
To be honest I don't know if we need libnghttp2.
A found this commit.
https://github.com/eclipse-che/che-operator/commit/432aef3d1a7112bb03dae88fbb8d1b0c4c00a8d6

Reply:
that was a cve update, so likely installed by some other rpm requirement. 

But I readded httpd-tools since we DO need that. Thanks for the review and catching my snafu :D